.github/workflows/push-tag-create-release.yml

```diff
@@ -101,10 +103,10 @@ jobs:
           path: reports
           retention-days: 7
 
-      # Generate changelog
-      - name: Generate changelog
-        # This changelog is only for client patch releases and build-tools releases
-        if: (fromJson(env.RELEASE_JSON).releaseType == 'patch' && fromJson(env.RELEASE_JSON).packageOrReleaseGroup == 'client') || fromJson(env.RELEASE_JSON).packageOrReleaseGroup == 'build-tools'
+      # Generate release notes
+      - name: Generate patch release notes
+        # This content is only for patch releases
+        if: fromJson(env.RELEASE_JSON).releaseType == 'patch'
```

Comment:
Will this now run for patch releases of anything including Independent packages and server? Also, build-tools used to get this step for every kind of release, now it'll be only for its patch releases, and we're explicitly excluding it from the next step, so there won't be auto-generation of any kind for build-tools minors and majors?

Reply:
> Will this now run for patch releases of anything including Independent packages and server?

Yes. However, I updated it to leave every release except client/server as draft.

> Also, build-tools used to get this step for every kind of release, now it'll be only for its patch releases, and we're
> explicitly excluding it from the next step, so there won't be auto-generation of any kind for build-tools minors and
> majors?

The workflow will fail if the release notes are missing, so I was just going to let them fail. But instead I inverted
the logic from the major/minor releases in this one, so now one of the two steps will be executed on each workflow run.